packages/cinterop/src/commonMain/kotlin/io/realm/internal/interop/sync/AuthProvider.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2021 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.internal.interop.sync
+
+@Suppress("filename")
```

Comment:
What is this for?

Reply:
I tried to follow the convention from the non-sync enums and define them in one file, but linting didn't allow naming the file differently than the class, when there was only one top level definition, so I tried to suppress that. It didn't work our so I just renamed the file ... and forgot to remove the annotation. Will do.